index.html

```diff
@@ -89,10 +89,245 @@ <h2 class="connect">
                 ></a>
               </li>
             </ul>
-
+          </div>
+        </div>
+      </section>
+      <section class="works">
+        <div class="work-item">
+          <div>
+            <img
+              class="images"
+              src="./assets/project_img1.png"
+              alt="project_img1"
+            />
+          </div>
```

Comment:
**Please follow the steps of styling in image below**
### Step 1
![re-review-third](https://user-images.githubusercontent.com/93039359/176534492-9e290bb8-ebb9-4f00-bf16-d68494d7915b.png)
### Step 2

![re-review-third part 2](https://user-images.githubusercontent.com/93039359/176534505-9375fc3b-75f2-4f67-8322-83f133a32959.png)

### After making the above changes your project will look like this
![re-review-third-project](https://user-images.githubusercontent.com/93039359/176534518-f3c7b0a6-fe62-48a1-bd42-1686bd137a78.png)

- [ ] Please make above changes because your project is not following the Figma template, :blush:

Reply:
@HAMMAS-SALEEM This breaks the code on the mobile view. Thank you